static/src/js/components/SearchPanels/SearchPanels.jsx

```diff
@@ -180,10 +181,16 @@ class SearchPanels extends PureComponent {
 
     const {
       pageNum: collectionsPageNum = 1,
-      sortKey: collectionsSortKey = collectionSortKeys.scoreDescending
+      paramCollectionSortKey: urlCollectionsSortKey
```

Comment:
is there a reason we are aliasing here to urlCollectionsSortKey?

Reply:
Yeah, look on line 294. We are updating the value of paramCollectionSortKey.